portfolio.html

```diff
@@ -0,0 +1,105 @@
+<!Doctype html>
+<html lang="en">
+    <head>
+        <title>Portfolio page</title>
+        <meta name="viewport contents" content="device-width, initial-scale=1.0">
+        <link rel ="stylesheet" type="text/css" href="portfolio-style.css">
+        <link rel ="stylesheet" type="text/css" href="normalise.css">
+    </head>
+    <body>
+        <header>
+
+            <section class="top-bar">
+                <div class="name">
+                    <h3>Matthew Hammond</h3>
+                </div>
+                <div class="top-links">
+                    <a href="#bio"><button type="button">Bio</button></a>
+                    <a href="#portfolio"><button type="button">Portfolio</button></a>
+                    <a href="#contact"><button type="button">Contact me</button></a>
+                </div>
+            </section>
+
+            <section class="landing">
+                <div class="landing-container">
+                    <img class="landing-image" src="./Assets/aerial.jpg">
+                    <div class="hero">
+                        <h1>Matthew Hammond</h1>
+                    </div>
+                    <div class="squire">
+                         <h1>Full-stack developer</h1>
+                    </div>
+                </div>
+            </section>
+            <!-------------------------------------------->
+            <section class="bio-container">
+                <div class="background" id="bio">
+                    <div class="sunset-container">
+                        <img class="sunset-image" src="./Assets/MOON.jpeg">
+                    </div>
+                        <div class="bio-title-container">
+                            <h1 class="Bio">Bio</h1>
+                        </div>
+                        <div class="paragraph-container">
+                            <p class="paragraph">
+                                Hey there. I'm currently learning to be a full-stack web-developer at Mayden academy.
+                                I consider my self a driven and motivated individual, who enjoys creative capacity and
```

Comment:
myself is one word
"who enjoys creative capacity" - I don't know what this means, "who enjoys working in a creative capacity" maybe?